web/server.go

```diff
@@ -1,3 +1,34 @@
 // Package web provides managing HTTP protocol.
 // Implement CRUD operations for filters over HTTP API.
 package web
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/lvl484/positioning-filter/repository"
+)
+
+type WebServer struct {
+	server *http.Server
+}
+
+func NewWebServer(filters repository.Filters, addr string) *WebServer {
+	router := newRouter(filters)
+	return &WebServer{
+		&http.Server{
+			Addr:    addr,
+			Handler: router,
+		},
+	}
+}
+
+func (ws *WebServer) Close() error {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
That's enough to use  context.Background() in shutdown, you don't need cancel here.

Reply:
fixed)